react-native-scripts/README.md

```diff
@@ -7,3 +7,21 @@ The README for this package is hosted at https://github.com/react-community/crea
 `yarn && yarn start` will start a watcher that will build artifacts and place them in the build directory.
 
 *Warning*: if you change the template project's dependencies, make sure to update `checkAppName` in `create-react-native-app/src/index.js`.
+
+## Adding Flow
```

Comment:
@dikaiosune entirely my bad. Understood your meaning but copied it to the wrong file (it *was* looking oddly empty... :))

I `--amend`ed the last commit to move it to the correct file.